packages/ts-sdk-apm/test/apm.test/apm.test.ts

```diff
@@ -41,19 +41,6 @@ describe('apm', () => {
 
       await telemetryMiddleware(next)(request)
     })
-
-    test('should use default apm and tracing configurations', async () => {
```

Comment:
If I may ask, why was this test removed?

Reply:
it was failing because we didn't have default values. Let me try to add it again.